enhancements/monitoring/scrape-profiles.md

```diff
@@ -0,0 +1,274 @@
+authors:
+  - @JoaoBraveCoding
+reviewers: # Include a comment about what domain expertise a reviewer is expected to bring and what area of the enhancement you expect them to focus on. For example: - "@networkguru, for networking aspects, please look at IP bootstrapping aspect"
+  - @openshift/openshift-team-monitoring
+approvers:
+  - TBD
+api-approvers: # In case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers). If there is no API change, use "None"
+  - TBD
+creation-date: 2022-12-06
+last-updated: yyyy-mm-dd
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - https://issues.redhat.com/browse/MON-2483
+see-also:
+  - "/enhancements/this-other-neat-thing.md"
+replaces:
+  - "/enhancements/that-less-than-great-idea.md"
+superseded-by:
+  - "/enhancements/our-past-effort.md"
+---
+
+# Scrape profiles
+
+## Summary
+
+The core OpenShift components ship a large number of metrics. A 4.12-nightly
+cluster on AWS currently produces around 350K series by default, and enabling
+additional add-ons increases that number. Users have repeatedly asked for a supported
+method of making Prometheus consume less memory, either by increasing the scraping
+interval or by scraping fewer targets -- for example, modifying the ServiceMonitors
+to drop metrics undesired to some users.
+
+These modifications are currently not possible, because the service monitors deployed to OCP
+are managed by operators and cannot be modified. This proposal outlines a solution for allowing
+users to set a level of scraping aligned with their needs.
+
+## Motivation
+
+OpenShift is an opinionated platform, and the default set of metrics has of course 
+been crafted to match what we think the majority of users might need. Nevertheless,
+users have repeatedly asked for the ability to reduce the amount of memory consumed by
+Prometheus either by lowering the Prometheus scrape intervals or by modifying ServiceMonitors.
+
+Users currently can not control the ServiceMonitors scraped by Prometheus since some of the
+metrics collected are essential for other parts of the system to function propperly
+(console, HPA and alerts). Users also are not allowed to tune the interval at which Prometheus
+scrapes targets as this again can have unforeseen results that can hinder the platform, a very
+low cadence may overwhelm the platform Prometheus instance a very high interval may render some
+of the default alerts ineffective.
+
+The goal of this proposal is to allow users to pick their desired level of scraping while limiting
+the impact this might have on the platform, via resources under the control of the
+cluster-monitoring-operator and other platform operators.
+
+
+### User Stories
+
+- As an OpenShift user, I want to lower the amount of memory consumed by Prometheus in a supported way, so I can choose between different scrape profiles, e.g `full` or `minimal`.
```

Comment:
The persona would rather be the OpenShift cluster administrator. Also "resources" rather than "memory" since CPU is also impacted?